.gitignore

```diff
@@ -221,6 +221,7 @@ vda/ssim
 vda/vda
 vda/vdad
 /win_build/*.sdf
+/win_build/x64
```

Comment:
this is wrong: `x64` folder should be inside `Build` folder

Reply:
Removed.